### PR TITLE
Update verida_network.md

### PR DESCRIPTION
**Pull request by ysam21**
Added expected delivery date of Milestone 1

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [x] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

https://github.com/w3f/Grants-Program/pull/887

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the update, but could you provide a recent estimate of the new delivery date and not something that is already in the past: " [Expected delivery June 2022]"

**Comment by Noc2** [comment]
Also, could you sign the latest version of our terms and conditions? See CLAassistant above

**Comment by ysam21** [comment]
Apologies @Noc2 , it was meant to say June 2023. I've updated the application.
 I've signed the CLA too.

File: applications/verida_network.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ See below for a summary of the software licencing for deliverables on the roadma
 
 ### Milestone 1 - Single Sign-On for Apps in Polkadot Ecosystem
 
-- **Estimated Duration:** 2 months
+- **Estimated Duration:** 2 months [Expected delivery June 2022]
```
Comment (alxs) [inline applications/verida_network.md:240]:
```suggestion
- **Estimated Duration:** 2 months [Expected delivery June 2023]
```

**Review by alxs — APPROVED** [review]

**Review by takahser — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Comment by Noc2** [comment]
Thanks for the update

**Review by dsm-w3f — APPROVED** [review]

**Review by gautamdhameja — APPROVED** [review]

**Pull request merged**